backend/user-service/app/migrate-and-start-dev.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+dotenv -e .env.development -- npx prisma generate
+dotenv -e .env.development -- npx prisma db push
```

Comment:
yup, the server can run but when it tries to read/write to a table in the db it fails because the schema was not pushed up to the db as `npx prisma db push` was not run

Reply:
also the database url needed to connect via the docker service name (`user-service-db`) instead of via `localhost`

then also need `directConnection=true` to avoid the issue with transactions not supported